project.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "workspace",
+	"$schema": "node_modules/nx/schemas/project-schema.json",
+	"targets": {
+		"local-registry": {
+			"executor": "@nx/js:verdaccio",
```

Comment:
So the local registry uses verdaccio? That's pretty clever tbh.

Reply:
All Nx here, so no idea what the inner workings are but glad you're happy :D